baseplate/experiments/providers/simple_experiment.py

```diff
@@ -27,6 +27,56 @@
 }
 
 
+def _generate_overrides(override_config):
+    """Generate a dictionary of overrides.
+
+    The format of the override config is expected to be a list of dicts,
+    where each dict contains a single key (the treatment to assign) and value
+    (the targeting tree for that treatment).
+
+    This combination is used to ensure that the ordering of the overrides is
+    maintained, allowing consistent overriding for any overlapping area
+    across targeting trees. Each incoming variant request will check each
+    targeting override in sequence until one is matched or none match.
+
+    If the override configuration does not contain a wrapping list (ie, it
+    is just a dictionary), then overrides will work, but override ordering
+    is not guaranteed. This means that if there is overlap between overrides
+    (ie, a particular user is eligible for two overrides), then the results
+    may not be consistent.
+
+    In the event of an invalid override:
+    1) if a dict of variant : target is not provided, that list item will be
+        ignored. All other overrides will still remain active.
+    2) if the targeting object is invalid, the override will remain active,
+        but will return false for all targeting
+
+    Multiple targets can be provided for a single variant.
+    """
+
+    override_list = []
+
+    if not override_config:
+        return None
+
+    if not isinstance(override_config, list):
+        if override_config:
```

Comment:
this if isn't necessary anymore since we're already bailing earlier on now.

Reply:
removed